geniepy/src/geniepy/datamgmt/scrapers.py

```diff
@@ -148,6 +151,118 @@ def scrape(self, chunksize: int, **kwargs) -> Generator:
         raise NotImplementedError
 
 
+class CitationScraper(BaseScraper):
+    """
+    Implementation of PubMed Articles Scraper.
+
+    https://www.ncbi.nlm.nih.gov/pubmed/
+    https://www.nlm.nih.gov/bsd/licensee/elements_descriptions.html
+
+    """
+
+    # Constants
+    API_BASE_URL = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&linkname=pubmed_pubmed_citedin"  # noqa
+    API_ID_PARAM = "&id="
+    API_KEY_PARAM = "&api_key="
+    API_KEYS = ["70b53d4e84436970587aef3493a723cae708",
```

Comment:
If we get to it, I think it's a good idea to put these keys in the configuration file so users and use their own - Can you make a reminder of this?